CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Notable changes to Pisoni will be tracked in this document.
 
+## 1.26.0 - 2019-03-05
+
+### Added
+
+- The stats of a service can now be deleted. [#19](https://github.com/3scale/pisoni/pull/19)
+
+### Changed
+
+- Ruby 2.2 is no longer supported. The minimum version is 2.3.0. [#18](https://github.com/3scale/pisoni/pull/18)
+
+
```

Comment:
extra blank line

Reply:
I see. I normally separate versions with 2 spaces but I'll change this to be consistent with what's already in the doc.